app/assets/javascripts/controllers/ops/pglogical_replication_form_controller.js

```diff
@@ -228,9 +228,10 @@ ManageIQ.angular.app.controller('pglogicalReplicationFormController', ['$http',
       data["user"] = subscription.user;
       data["password"] = subscription.password;
       data["port"]     = subscription.port;
+      data["id"] = subscription.id
     }
     miqService.sparkleOn();
-    var url = '/ops/pglogical_validate_subscription/' + pglogicalReplicationFormId;
+    var url = '/ops/pglogical_validate_subscription'
```

Comment:
@carbonin @yrudman this change should be ok, as pglogical_validate_subscription method is not expecting pglogicalReplicationFormId to be sent up with params. @carbonin did you test this in UI just to make sure?